### ISSUE TITLE
Regarding the number of submissions

### ISSUE THREAD

**Issue by junjun621** [PRBonn/semantic-kitti-api#165]
Hello, I tried to submit new test results today, but got this message:The maximum number of submissions has been reached. What should I do to get the test results?
Thank you very much for your reply！

**Comment by jbehley** [comment]
That's by design to avoid overfitting on the test results as also described on the competition website. Ablations and hyperparameter tuning should be only conducted on the validation set as it is good practice to obtain unbiased test results.
